easybuild/easyconfigs/i/IMB/IMB-4.0.1-intel-2015a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'MakeCp'
+
+name = 'IMB'
+version = '4.0.1'
+
+homepage = 'https://software.intel.com/en-us/articles/intel-mpi-benchmarks'
+description = """The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and
+ global communication operations for a range of message sizes"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://software.intel.com/sites/default/files/managed/34/aa/']
+sources = ['%(name)s_%(version)s.tgz']
+
+prebuildopts = "cd src && "
+buildopts = "all"
```

Comment:
no, building IMB on top of Intel MPI works, it uses the `make_ict` (intel cluster tools) Makefile by default, which uses `mpiicc` (as opposed to `mpicc`)